.cypress/integration/notebooks_test/notebooks.spec.js

```diff
@@ -31,10 +25,18 @@ const moveToPanelHome = () => {
   cy.visit(`${Cypress.env('opensearchDashboards')}/app/observability-dashboards#/`);
 };
 
+const moveToNotebookHome = () => {
```

Comment:
moveToNotebookHome seems identical to moveToTestNotebook without the timeout, are they both necessary?

Reply:
`moveToNotebookHome` -> Loads notebooks home page 
`moveToTestNotebook` -> Loads the test notebook
Both are intended for different types of page loads.